samples/boards/microbit/pong/README.rst

```diff
@@ -0,0 +1,17 @@
+.. _microbit_pong:
+
+BBC micro:bit pong game
+#######################
+
+Overview
+********
+
+Play pong over Bluetooth between two micro:bit devices.
```

Comment:
Could use more information about the sample.  For example, you should add what you mention in the commit message:

   This pong game using the display on the micro:bit
   and its two buttons. This version currently supports a solo game where the
   ball bounces from the "roof" and lets the player controls both paddles.

You also talk about bluetooth.  How that's involved in this sample?